package.json

```diff
@@ -31,10 +31,11 @@
     "react-countup": "4.1.3",
     "react-dom": "16.8.4",
     "react-helmet": "^5.0.0",
+    "react-markdown": "^v4.3.1",
     "react-resize-detector": "1.1.0",
     "recharts": "^1.1.0",
-    "react-markdown": "^v4.3.1",
     "reconnecting-websocket": "^v3.2.2",
```

Comment:
reconnecting-websocket is still required?

Reply:
Thank you for your reminder, I need to delete it

Reply:
We can fix it later :+1: